src/otx/v2/adapters/datumaro/adapter/anomaly_dataset_adapter.py

```diff
@@ -9,22 +9,16 @@
 
 import cv2
 import numpy as np
-from datumaro.components.dataset import Dataset as DatumaroDataset
+from datumaro.components.annotation import Bbox as DatumBbox
+from datumaro.components.annotation import Label as DatumLabel
+from datumaro.components.annotation import Mask as DatumMask
+from datumaro.components.dataset import Dataset as DatumDataset
```

Comment:
Just out of curiosity, can't we use Datumaro like we do in numpy?
```python
import datumaro as dm
dm.Dataset.import_from(...)
```
And, is it a convention to use `DatumXXX` instead of `DatumaroXXX`?

Reply:
Thank you for the good question. I will fix this Datum prefix when removing all annotation entities within otx.api. At that time, I am going to use such as `from datumato.components.annotation import Bbox, Label, Mask`.
Of course, it is okay to use `dm.Label...`.